k8s/upgrades/0.9.0-1.0.0/jiva/jiva_volume_upgrade_auto.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+source ../util.sh
+
+mkdir upgrade_tmp
+
+function on_exit() {
+    echo "Clearing temporary files"
+    rm -r upgrade_tmp
+}
+trap 'on_exit' EXIT
+
+upgrade_version="1.0.0"
+current_version="0.9.0"
+
+export upgrade_version
+export current_version
+
+function usage() {
+    echo
+    echo "Usage:"
+    echo
+    echo "$0 <pv-name>"
+    echo
+    echo "  <pv-name> Get the PV name using: kubectl get pv"
+    exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+pv=$1
+upgrade_task=$2
+export pv
+export upgrade_task
+
+# Check if pv exists
+get_status=$(kubectl get pv "$pv" 2>&1);check_pv=$?
+if [ $check_pv -ne 0 ]; then
+    reason=$(echo $get_status | tr --delete ":")
+    patch_upgrade_task_error "$upgrade_task" "PRE_UPGRADE" "failed to upgrade $pv" "$reason"; 
+    exit 1
+fi
+
+# Check if CASType is jiva
+cas_type=$(kubectl get pv "$pv" -o jsonpath="{.metadata.annotations.openebs\.io/cas-type}")
+if [ "$cas_type" != "jiva" ]; then
+    reason=$(echo $get_status | tr --delete ":")
+    patch_upgrade_task_error "$upgrade_task" "PRE_UPGRADE" "failed to upgrade $pv" "invalid castype $cas_type"; 
+    exit 1
+fi
+
+ns=$(kubectl get pv "$pv" -o jsonpath="{.spec.claimRef.namespace}")
```

Comment:
I think we need take namespace as a argument (or) from sc we need to know where jiva deployment pods are running.
Refer: 
https://github.com/openebs/maya/pull/1183

Reply:
We didn't handled in upgrade case but now when we are coming from MO need to fix it

Reply:
Done